src/client/tab.rs

```diff
@@ -656,6 +656,11 @@ impl Tab {
     pub fn toggle_fullscreen_is_active(&mut self) {
         self.fullscreen_is_active = !self.fullscreen_is_active;
     }
+    pub fn set_force_render(&mut self) {
```

Comment:
Nice. I would have gone with something on the state of `Tab` that bypasses the `should_render` of each pane, but the more I think about it, the more I like this solution.